src/main/java/it/gov/pagopa/payment/connector/rest/paymentInstrument/PaymentInstrumentRestConnector.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.payment.connector.rest.paymentInstrument;
+
+import it.gov.pagopa.payment.dto.PinBlockDTO;
+import it.gov.pagopa.payment.dto.VerifyPinBlockDTO;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Service
+public interface PaymentInstrumentRestConnector {
+    VerifyPinBlockDTO checkPinBlock(@RequestBody PinBlockDTO pinBlockDTO, String userId);
```

Comment:
this annotation is useless here
```suggestion
    VerifyPinBlockDTO checkPinBlock(PinBlockDTO pinBlockDTO, String userId);
```